Sources/InternxtSwiftCore/Services/Network/NetworkFacade.swift

```diff
@@ -310,7 +326,7 @@ public struct NetworkFacade {
     }
     
     class UploadPartOperation: AsyncOperation {
-        let encryptedChunk: Data
+        var encryptedChunk: Data?
```

Comment:
Why does this need to be changed to `var`?

Let's avoid overwriting a variable's value to handle binary data as that could lead to unintended consequences like corruption of the data, data loss, etc.